app/src/main/java/com/willowtree/vocable/keyboard/KeyboardViewModel.kt

```diff
@@ -31,14 +31,13 @@ class KeyboardViewModel(numbersCategoryId: String, mySayingsCategoryId: String)
             val mySayingsPhrases =
                 presetsRepository.getPhrasesForCategory(mySayingsCategoryId)
             with(presetsRepository) {
-                // TODO: Use currently set Locale
                 addPhrase(
                     Phrase(
                         phraseId,
                         System.currentTimeMillis(),
                         true,
                         System.currentTimeMillis(),
-                        mapOf(Pair(Locale.US.language, phraseStr)),
+                        mapOf(Pair(Locale.getDefault().toString(), phraseStr)),
```

Comment:
Confused on what you're passing in here. Are you converting this pair into a map? Pairs are pretty expensive so I try to avoid if I can. Would this be the same if you had kotlin extensions to do `Locale.getDefault().toString() to phaseStr`?

Reply:
Yeah if you call `mapOf(Pair(...))`, it will convert it into a map with `pair.first` being the key and `pair.second` being the value.
I think `Locale.getDefault().toString() to phaseStr` also creates a Pair

Reply:
Is it less expensive to do something like `mutableMapOf().apply { put(Locale.getDefault.toString, phraseStr) }`?

Reply:
Ahh, not sure but I doubt it. In any case, I thought the `to` thing returned a map forgot it was a pair. This is probably a case of premature optimization. I'd look into if it's possible to avoid creating pairs if possible with an alternative data structure but I think this makes it more readable at least.